day_09.md

```diff
@@ -0,0 +1,265 @@
+# [A walk in JavaScript](/README.md)
+
+## DAY 9
+
+- Asynchronous programming
+  - Event Loop
+  - Callback
+  - Promises
+  - Async/Await
+  - Exercises
+
+## Asynchronous programming
+
+Until ES6 (we'll get into that later, but we already saw a glimpse of that with Generators) there was nothing in the JS core specifying asynchronism, it had a fire-heated mark with the words "run-to-completion". In fact all asynchronous behavior depended on the host environment providing some kind of API in order to handle the asynchronism on it's side and communicate to the JavaScript engine the outcome if necessary.
+
+For example, you will find references and definitions of **timers** on the MDN documentation for [WindowOrWorkerGlobalScope](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope) describing them in the context of a browser host environment but if you make a search of **timer** or **timeout** in the [ES6](https://www.ecma-international.org/ecma-262/6.0/) spec you'll find ... absolutely nothing! So what does it mean? Plain a simple, it means **JavaScript has no notion of timers**, the host does, and as we discussed in previous chapters, a web browser is not the only host for JavaScript, it's just one of many AND most likely this very environment host doesn't even have the code to perform the asynchronism, it acts just as a bridge to their own host which might happen to be an operative system!!!!
```

Comment:
**JavaScript has no notion of timers** sounds a little weird. May be you can change it for **JavaScript is not aware of** or **JavaScript doesn't have notion of timers**

Reply:
maybe I wasn't clear. My intention was to make clear "the notion of timers" can't be found in javascript, but is aware of timers through the host's API